tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -112,6 +113,17 @@ def _forward_backward_test_impl(
                 deallocate_pipeline_output=deallocate_pipeline_outputs,
             )
 
+            h_size = PipelineParallelForwardBackwardTest.HIDDEN_SIZE
+            m_size = PipelineParallelForwardBackwardTest.MICRO_BATCH_SIZE
+            target_loss = 0
+            for r in range(self.world_size):
+                target_loss = target_loss + h_size*(r + 1) + 1
+            target_loss = target_loss * (h_size ** 2) * m_size / self.world_size
+
+            for loss_item in loss:
+                x = loss_item['avg']
+                self.assertTrue(torch.allclose(x, target_loss*torch.ones_like(x)))
```

Comment:
nit-picking

```suggestion
                torch.testing.assert_close(x, target_loss*torch.ones_like(x))
```

ref: https://pytorch.org/docs/stable/testing.html#torch.testing.assert_close